index.html

```diff
@@ -21,17 +21,19 @@
     The release below contains <a href="/2020-02-18-bitcoin-abc-0.21.0/">protocol upgrades</a> scheduled to activate on May 15th, 2020.
     Update early to keep your node in sync with the Bitcoin Cash network.
     <br><br>
-    <span style="font-weight: bold">Install the latest version (0.21.4):</span>
+    <span style="font-weight: bold">Install the latest version:</span>
     <br>
     <a style="margin-right: 1em;" href="https://github.com/Bitcoin-ABC/bitcoin-abc">Source Code</a>
-    <a style="margin-left: 1em;" href="https://download.bitcoinabc.org/">Binaries</a>
+    <a style="margin-left: 1em;" href="https://download.bitcoinabc.org/latest/">Binaries</a>
     <br>
     <span style="font-weight: bold">Packages:</span>
     <br>
     <a style="margin-right: 1em;" href="https://hub.docker.com/r/zquestz/bitcoin-abc">Docker</a>
     <a style="margin-left: 1em;" href="https://aur.archlinux.org/packages/bitcoin-abc/">Arch Linux</a>
     <a style="margin-right: 1em;" href="https://aur.archlinux.org/packages/bitcoin-abc-qt/">(Qt)</a>
     <a style="margin-left: 1em;" href="https://launchpad.net/~bitcoin-abc/+archive/ubuntu/ppa">Ubuntu</a>
+    <br><br>
+    <span>Past binary version are available <a href="https://download.bitcoinabc.org/">here</a></span>
```

Comment:
version should be plural here.

But I even wonder if this part is necessary... it's easy to navigate to the prior binaries from the /latest link, and is there really any value added to put this link prominently on the front page?